series/series_test.go

```diff
@@ -94,6 +95,18 @@ func TestSeries_Compare(t *testing.T) {
 			Bools([]bool{true, false, true}),
 		},
 		{
+			Times([]time.Time{time.Now(), time.Now().Add(time.Minute * 2)}),
```

Comment:
This is extremely dangerous! `time.Now()` will change on every call! Parse a fixed date from a `string` instead so that every time tested is the same